definition/data/sheets/CaseEventToFields.json

```diff
@@ -28,11 +30,11 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewOrder", "CaseFieldID": "draftOrderDoc", "PageID": 1, "DisplayContext": "READONLY", "PageFieldDisplayOrder": 1},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewOrder", "CaseFieldID": "assignedTo", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "directionList", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 1, "CallBackURLMidEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/mid", "RetriesTimeoutURLMidEvent": "5,10,15"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "eyewitnessUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "eyewitnessUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 3, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "extraDocUploadList", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 4, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "docUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 5, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "extraDocUploadList", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "docUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 3, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "docUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 6, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
```

Comment:
to make these items in sequence in this file as well, should't we move this after `eyewitnessUploadForParty`?

Reply:
yes, done